Libraries/src/Amazon.Lambda.Annotations.Models/Amazon.Lambda.Annotations.Models.csproj

```diff
@@ -1,14 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AssemblyVersion>1.5.2</AssemblyVersion>
+    <RootNamespace>Amazon.Lambda.Annotations</RootNamespace>
     <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
 
     <!--This assembly contains some internal methods that need to be accessed by the Amazon.Lambda.Annotations.SourceGenerator assembly.
     Both these assemblies need to be strongly signed for the InternalsVisibleTo attribute to take effect.-->
     <AssemblyOriginatorKeyFile>..\..\..\buildtools\public.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    
```

Comment:
Because we want to make sure to never generate a package for this project can you add `<IsPackable>false</IsPackable>`.

Reply:
Done